src/main/java/com/rackspace/ceres/app/services/MetadataService.java

```diff
@@ -214,6 +224,42 @@ public MetadataService(ReactiveCqlTemplate cqlTemplate,
         .flatMapMany(Flux::fromIterable);
   }
 
+  public Flux<TsdbQuery> getMetricsAndTagsAndMetadata(List<TsdbQueryRequest> queries, List<Granularity> granularities) {
```

Comment:
With reference to my comment in https://github.com/racker/ceres/pull/34/files#r607837075 I think the logic will be change here too.

Reply:
done